content/en/blog/introduction_to_wamr_running_modes/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Introduction to WAMR running modes"
+description: ""
+excerpt: ""
+date: 2023-01-19T17:20:07+08:00
+lastmod: 2023-01-28T17:20:07+08:00
+draft: false
+weight: 50
+images: []
+categories: ['introduction']
+tags: []
+contributors: ['Tianlong Liang']
+pinned: false
+homepage: false
+---
+
+When you first meet WAMR, want to give it a taste, either using `iwasm` to run a hello world wasm application, or embed a WAMR into your hello world program, I bet you won't even sweat a bit. But when you're up to do something serious, maybe use it for real world applications and start to think about performance and response time, things changes. At this time, the different running modes (AOT, interpreter, fast interpreter, fast JIT, LLVM JIT, multi-tier JIT) we proudly made, aiming to provide user more adaptability and flexibility, could backfire and encumber user.
```

Comment:
`or embed` -> `or embedding`
`maybe use it` -> `maybe to use it`?
How about change `the different running modes` to `the rich running modes`

Reply:
`interpreter, fast interpreter` -> `Classic Interpreter, Fast Interpreter`
`fast JIT` -> `Fast JIT`
`multi-tier JIT` -> `Multi-tier JIT`

Just be same as below